server/setup_flow.go

```diff
@@ -208,10 +208,10 @@ func (p *Plugin) stepChooseEdition() flow.Step {
 
 func (p *Plugin) stepServerAddAppLink() flow.Step {
 	return flow.NewStep(stepServerAddAppLink).
-		WithPretext("##### :white_check_mark: Step 2: Configure Mattermost Application Link in Jira").
+		WithPretext("##### :white_check_mark: Step 2: Configure the Mattermost Application Link in Jira").
```

Comment:
This step 2 text should match the one on line 108 (assuming they're the same step). With this in mind, which version of step 2 text is the one we'll keep? Which one is more representative and describes what the user is trying to achieve?